api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2027,6 +2045,15 @@ service AuthService {
   // UpsertClusterAlert creates a cluster alert.
   rpc UpsertClusterAlert(UpsertClusterAlertRequest) returns (google.protobuf.Empty);
 
+  // AcknowledgeAlert marks a cluster alert as acknowledged.
+  rpc AcknowledgeAlert(types.AlertAcknowledgement) returns (google.protobuf.Empty);
```

Comment:
Nit: Part of me thinks that we could just call this `CreateAlertAck` for consistency with the naming of other CRUD methods and the other AlertAck messages..